Neos.RedirectHandler/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "neos/redirecthandler",
+    "type": "neos-package",
+    "description": "Basic API to handle http redirect with Flow Framework",
+    "license": "GPL-3.0+",
+    "require": {
+        "typo3/flow": "~3.2"
```

Comment:
not sure if this would have unwanted side effects like not being able to use Neos master with Flow master